nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -385,6 +388,89 @@ public void testDropTable() throws IOException {
     verifyCommitMetadata();
   }
 
+  private void testRegister(TableIdentifier identifier, String metadataVersionFiles) {
```

Comment:
This is a utility method, but it is names like a test method... Would you mind renaming to something like `validateRegister`?

Reply:
updated...thank you